.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+    
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+
+  test:
+    name: Tests
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.7"
+    
+    env:
+      PUPPET_VERSION: "7"
```

Comment:
Right now we only support Puppet 7, but we want to support Puppet 8 as well. Please include the Puppet version in the test matrix.